homeassistant/components/cover/homematic.py

```diff
@@ -69,3 +71,39 @@ def _init_data_struct(self):
         """Generate a data dictoinary (self._data) from metadata."""
         self._state = "LEVEL"
         self._data.update({self._state: STATE_UNKNOWN})
+        try:
+            self._data.update(
+                {'LEVEL_2': self._hmdevice.get_cover_tilt_position()})
+        except:
+            pass
+
+    @property
+    def current_cover_tilt_position(self):
+        """Return current position of cover tilt.
+
+        None is unknown, 0 is closed, 100 is fully open.
+        """
+        if 'LEVEL_2' not in self._data:
+            return None
+
+        return int(self._data.get('LEVEL_2', 0) * 100)
+
+    def set_cover_tilt_position(self, **kwargs):
+        """Move the cover tilt to a specific position."""
+        if ATTR_TILT_POSITION in kwargs:
+            position = float(kwargs[ATTR_TILT_POSITION])
+            position = min(100, max(0, position))
+            level = position / 100.0
+            self._hmdevice.set_cover_tilt_position(level, self._channel)
```

Comment:
Don't use this function. The helper function are only for the library. Look if LEVEL_2 exists inside data and use the code to write that to xmlrpc.